src/main/java/duke/exceptions/EmptyDescriptionException.java

```diff
@@ -0,0 +1,16 @@
+package duke.exceptions;
+
+public class EmptyDescriptionException extends Exception {
+    protected String typeOfTask;
+
+    public EmptyDescriptionException (String typeOfTask) {
+        this.typeOfTask = typeOfTask;
+    }
+    public void printErrorMessage() {
+        if (typeOfTask.equals("marked") || typeOfTask.equals("unmarked")) {
+            System.out.println("OOPS!! Task to be " + typeOfTask + " was not specified!");
+        } else {
+            System.out.println("OOPS!! The description of " + typeOfTask + " cannot be empty!");
+        }
```

Comment:
Instead of checking whether it is marked or unmarked and proceed to print a different error message for it, you might want to just create another exception separately for the mark and unmark feature.